src/lightning_app/cli/commands/connection.py

```diff
@@ -47,51 +50,64 @@ def connect(app_name_or_id: str, yes: bool = False):
                 click.echo(f"You are already connected to the cloud Lightning App: {app_name_or_id}.")
         else:
             disconnect()
-            connect(app_name_or_id, yes)
+            connect(app_name_or_id)
 
     elif app_name_or_id.startswith("localhost"):
 
-        if app_name_or_id != "localhost":
-            raise Exception("You need to pass localhost to connect to the local Lightning App.")
+        with Progress() as progress_bar:
+            event = Event()
+            connecting = progress_bar.add_task("[magenta]Setting things up for you...", total=_PROGRESS_TOTAL)
+            thread = Thread(target=update_progresss, args=[event, progress_bar, connecting], daemon=True)
+            thread.start()
 
-        retriever = _LightningAppOpenAPIRetriever(None)
+            if app_name_or_id != "localhost":
+                raise Exception("You need to pass localhost to connect to the local Lightning App.")
 
-        if retriever.api_commands is None:
-            raise Exception(f"The commands weren't found. Is your app {app_name_or_id} running ?")
+            retriever = _LightningAppOpenAPIRetriever(None)
 
-        commands_folder = os.path.join(_LIGHTNING_CONNECTION_FOLDER, "commands")
-        if not os.path.exists(commands_folder):
-            os.makedirs(commands_folder)
+            if retriever.api_commands is None:
+                raise Exception(f"The commands weren't found. Is your app {app_name_or_id} running ?")
```

Comment:
What about something like:
"Connection wasn't successful. Is your app {app_name_or_id} running?"

Reply:
```suggestion
                raise Exception(f"Connection wasn't successful. Is your app {app_name_or_id} running?")
```